gateway-js/src/config.ts

```diff
@@ -20,7 +20,7 @@ export type Experimental_DidResolveQueryPlanCallback = ({
   readonly serviceMap: ServiceMap;
   readonly operationContext: OperationContext;
   readonly requestContext: GraphQLRequestContextExecutionDidStart<
-    Record<string, any>
+  Record<string, any>
```

Comment:
Prefer the indentation here, can we disable this rule?

Reply:
I agree that this is uglier, but if this rule is customizable for this specific case I can't figure it out and I think we do definitely want this rule in general. I think the best solution may be to just not put a line break in between <>. I'd be willing to bet without looking that this is one of the rules prettier disables so it can do its own thing